static/js/dashboard_routes.js

```diff
@@ -13,7 +13,7 @@ import Learner from './components/Learner';
 import OrderSummaryPage from './containers/OrderSummaryPage';
 
 const errorLoading = error => {
-  console.trace(error);
+  console.log("loading error caused by:", error)
```

Comment:
why console here?

Reply:
`console.trace` was showing the stack trace of `errorLoading`, which doesn't contain any useful information. Instead we want to see the stack trace of `error` which shows up via `console.log`